### PR TITLE
Github Action & Correct Dependency

### PR DESCRIPTION
**Pull request by sdimitro**
# Commit 1

While trying to run the test-suite of libkdumpfile I realized that we  miss one dependency. This commit adds that dependency so all the tests can build correctly.

# Commit 2

Adds two actions that build libkdumpfile in two ways. The first one builds the debian package very similary to how linux-pkg does it, then installs that debian file and tries it in test crash dump of sdb. The second action builds and installs the library the same way a developer would do when they're iterating on a change.

Note that this commit is not attempting to replace linux-pkg in building the debian package. This is mostly to guard against any changes that are introduced by PRs and could potentially break anything packaging-related. The point is to catch any failure before it propagates.

Besides, the current code itself I'm open to any feedback on structural changes like abstracting certain things out to make them more generic.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      # Note: The latest version of libkdumpfile has been 0.3.0
+      #       and has stayed such for many years, even though
+      #       the library has had multiple breaking changes. The
+      #       actual version number can be acquired through the
+      #       m4 macros defined in configure.ac (pkg_version_*).
+      #       Since there is not a clean/straightforward way to
+      #       get that value from the file we hardcode it here
+      #       for the changelog that we generate.
+      #
+      - run: dch --create --package "libkdumpfile" -v "0.3.0" "Auto-generated entry."
```
Comment (pzakha) [inline .github/workflows/main.yml:40]: I'd suggest to use that command manually to create a permanent changelog file and check it in. Actually, we should add changelog files to all of our packages.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+        with:
+          python-version: '3.6'
+      #
+      # Needed for all the debian file related utilities.
+      #
+      - run: sudo apt install debhelper devscripts equivs
```
Comment (pzakha) [inline .github/workflows/main.yml:23]: nit: `apt-get` is recommended over `apt` in scripts.
Also, might want to add `-y`

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      #
+      - run: dch --create --package "libkdumpfile" -v "0.3.0" "Auto-generated entry."
+      #
+      # Perform build and generate package.
+      #
+      - run: dpkg-buildpackage -b -us -uc
+      #
+      # Install the package.
+      #
+      - run: sudo dpkg -i ../libkdumpfile_*deb
```
Comment (pzakha) [inline .github/workflows/main.yml:48]: `dpkg -i` doesn't install any of the package's dependencies. I'd suggest to use `apt-get install` instead.

**Comment by sdimitro** [comment]
Because I added the changelog in the repo as a separate commit in the beginning of the PRs history, Github got bamboozled. New PR can be found here https://github.com/delphix/libkdumpfile/pull/9

**Pull request closed without merging**